appscale/tools/parse_args.py

```diff
@@ -625,19 +623,16 @@ def validate_infrastructure_flags(self):
         raise BadConfigurationException("--disks must be a dict, but was a " \
           "{0}".format(type(self.args.disks)))
 
-    if self.args.instance_type in EC2Agent.DISALLOWED_INSTANCE_TYPES and \
-      not (self.args.force or self.args.test):
-      LocalState.confirm_or_abort("The {0} instance type does not have " \
-        "enough RAM to run Cassandra in a production setting. Please " \
-        "consider using a larger instance type.".format(
-        self.args.instance_type))
+    if not self.args.instance_type:
+      raise BadConfigurationException("Cannot start a cloud instance without " \
+                                      "the Instance type.")
 
-    if self.args.gce_instance_type in GCEAgent.DISALLOWED_INSTANCE_TYPES and \
+    if self.args.instance_type in self.DISALLOWED_INSTANCE_TYPES and \
       not (self.args.force or self.args.test):
```

Comment:
Please add 2 more spaces before the beginning here, to distinguish the continuation line from the indented block underneath.